apps/prairielearn/src/ee/pages/institutionAdminCourseInstance/institutionAdminCourseInstance.ts

```diff
@@ -40,15 +63,34 @@ router.get(
 router.post(
   '/',
   asyncHandler(async (req, res) => {
+    const { course_instance } = await selectCourseInstanceAndCourseInstitution({
+      institution_id: req.params.institution_id,
+      unsafe_course_id: req.params.course_id,
+      unsafe_course_instance_id: req.params.course_instance_id,
+    });
+
     if (req.body.__action === 'update_enrollment_limit') {
       await queryAsync(sql.update_enrollment_limit, {
         institution_id: req.params.institution_id,
-        course_instance_id: req.params.course_instance_id,
+        course_instance_id: course_instance.id,
         enrollment_limit: req.body.enrollment_limit || null,
       });
```

Comment:
Now that we have `selectCourseInstanceAndCourseInstitution` above, we can probably omit the `institution_id` check here, as we've already verified that the course instance belongs to the institution.

Reply:
I changed this in 63a4c48.